scripts/prepareCrab.py

```diff
@@ -181,6 +186,9 @@ def makeSubmitFiles(inputFile, nThreads, submit, doConfig, dryRun, match_expr, v
         ' ex: 1 1 for all, 2 1 for every second file', default=(0,0))
 parser.add_argument('-j', '--nThreads', type=int, default=1, 
     help="number of threads (make sure its consistent if you're not regenerating configs)")
+parser.add_argument('-p', '--storage', default='/store/group/cmst3/group/wmass/w-mass-13TeV/NanoAOD', type=str, help='Storage path of output Ntuples(default CERN storage)')
```

Comment:
I imagine -p stands for path, but since the long version is "--storage" this would seem confusing for a user (in fact it is the same for -j/--nThreads which was already there, but I got used to it now).
Same with -t/--site.
For the options you added one could just leave the long name

Reply:
Edited by removing `-p/-t`.